lib/autobox/Transform.pm

```diff
@@ -74,6 +84,20 @@ particular when the values are hashrefs or objects.
 
     $books->uniq_by("id");
 
+    $books->order_by("name");
+    $books->order_by(name => "desc");
+    $books->order_by(price => "num");
+    $books->order_by(price => [ "num", "desc" ]);
+    $books->order_by(name => [ sub { uc($_) }, "desc" ]);
+    $books->order_by([ price_with_tax => $rate ] => "num");
```

Comment:
I don't quite get this one, what is $rate? just another hashref key of a single $book?